src/scripting/logichook/asserter/IntentConfidenceAsserter.js

```diff
@@ -0,0 +1,56 @@
+const _ = require('lodash')
+
+/**
+ * Can be used as local, and as global asserter
+ * @type {module.IntentConfidenceAsserter}
+ */
+module.exports = class IntentConfidenceAsserter {
+  constructor (context, caps = {}, globalArgs = {}) {
+    this.context = context
+    this.caps = caps
+    this.hasGlobalExpectedMinimum = typeof globalArgs.EXPECTED_MINIMUM !== 'undefined'
```

Comment:
Maybe use isNaN ? personally I don't like these "typeof" checks

And the globalArgs should not be expected to contain arg names all in capital - just use "expectedMinimum"

Reply:
done